source/beamline/IDEAS/IDEASBeamline.cpp

```diff
@@ -196,8 +175,6 @@ void IDEASBeamline::setupExposedDetectors()
 	addExposedDetector(ReferenceIonChamberScaler_);
 	addExposedDetector(ketek_);
 	addExposedDetector(ge13Element_);
-	addExposedDetector(ketekRealTime_);
```

Comment:
These removals (and the corresponding removals above of the exposed controls) is why we put the one off detectors inside the beamline class.  The exposed detectors and controls are incredibly important to running of generic scans and taking advantage of many of the "for free" features that AM provides.